picamera2/previews/drm_preview.py

```diff
@@ -206,8 +206,8 @@ def render_drm(self, picam2, completed_request):
         if self.overlay_fb is not None:
             width, height = self.overlay_fb.width, self.overlay_fb.height
             self.crtc.set_plane(self.overlay_plane, self.overlay_fb, x, y, w, h, 0, 0, width, height)
-        overlay_old_fb = None  # The new one has been sent so it's safe to let this go now
```

Comment:
Just curious as to what it objected to here...?

Reply:
I think that's because ```overlay_old_fb``` isn't read by anything.  But I see that you're using it to hold onto an object temporarily.